src/musicPlayer.java

```diff
@@ -0,0 +1,37 @@
+import components.queue.Queue;
+import components.queue.Queue1L;
+
+/** */
+public class musicPlayer {
+    /** */
+
+    private Queue<String> playlist;
+
+    /** */
+    public musicPlayer() {
+        this.playlist = new Queue1L<>();
+    }
+
+    /** */
+    public void queue(String song) {
```

Comment:
Are you planning to use Strings long term? I think it's fine if all you need is a string like: `blank by blank`. However, if you want more sophisticated song types, I might make an inner class called Song or use a map to allow for flexible song properties (e.g., title, artist, album, pathToFile, pathToCoverArt, etc.).

Reply:
I would also consider using method names that are more specific to the application. For example, rather than queue, I might propose the following:
```suggestion
    public void addSong(String song) {
```